lib/language.js

```diff
@@ -422,6 +473,7 @@ function init() {
       ,fr: 'Percentiles'
       ,pt: 'Percentis'
       ,ro: 'Grafic percentile'
+      ,бг: "Процентна графика"
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Mixed double and single quotes.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/819607010/issues/source?bid=2452379&fileBranchId=2748601#l476)